packages/synapse-react-client/src/components/AccessRequirement/AddConditionsForUseButton/AddConditionsForUseButton.stories.tsx

```diff
@@ -13,5 +13,6 @@ type Story = StoryObj<typeof meta>
 export const Demo: Story = {
   args: {
     entityId: 'syn45328519',
+    onACTMemberClick: fn(),
```

Comment:
In storybook 8, `on...` callbacks are no longer automatically mocked. We can manually mock them with `fn` from `@storybook/test`.